src/Practice2_Animal/Cat.java

```diff
@@ -0,0 +1,17 @@
+package Practice2_Animal;
+
+public class Cat extends Animal{
+	private String mauLong;
+	private int chieuCao;
+
+	public Cat(String ten, int tuoi, String giong, String mauLong, int chieuCao) {
+		super(ten, tuoi, giong);
+		this.mauLong = mauLong;
+		this.chieuCao = chieuCao;
+	}
+	public void Cat_info() {
+		super.Info();
+		System.out.println("màu lông: " + mauLong + ", chiều cao: " + chieuCao);
+	}
```

Comment:
ri thì đây là method riêng của class Cat chứ ko phải sự ghi đè từ việc mở rộng method đã có sẵn của class cha Animal nên chỗ này ko hợp lý nhé thanh niên 
```suggestion
        @Override
	public void displayInfo() {
		super.displayInfo();
		System.out.println("màu lông: " + mauLong + ", chiều cao: " + chieuCao);
	}
```